AxonDeepSeg/__init__.py

```diff
@@ -1 +1,7 @@
+import os
+import sys
+
+# make sure repo directory (two directories up) is on path (to access config.py)
+sys.path.append(os.path.abspath(os.path.join(__file__, os.pardir, os.pardir)))
```

Comment:
Note: We try to access a file outside of the `AxonDeepSeg` package directory (`config.py`), but there is no root-level `__init__.py`. 

On Windows, the root of the repo doesn't get added to the PATH, yet on Linux/macOS it does. This caused test failures, hence me adding this line.

But, I'm surprised! I wonder why these errors didn't occur with the old installation method.